bb/src/main/java/com/richardmcdougall/bb/BurnerBoardUtil.java

```diff
@@ -23,5 +33,43 @@
             BOARD_ID = Build.MODEL;
         }
     }
-}
 
+    // Switch any of these to 'true' to force identification as that board type.
+    // Only useful for debugging! -jib
+    private static boolean kForceBBTypeAzul = false;
+    private static boolean kForceBBTypeClassic = false;
+    private static boolean kForceBBTypeDirectMap = false;
+    private static boolean kForceBBTypeMast = false;
+    private static boolean kForceBBTypePanel = false;
+
+    /*
+        XXX TODO: Azul is currently the 'fall through' case in BBService. There's probably a positive way to
+        identify an Azul board. -jib
+     */
+    public static final boolean isBBAzul() {
```

Comment:
Can we just check boardtype for Azul?

Reply:
Yup - that's why i asked you on friday, will update.